### ISSUE TITLE
Inquiry About Using Lag-Llama Model for Customer Trend Analysis

### ISSUE THREAD

**Issue by RakshitAralimatti** [time-series-foundation-models/lag-llama#96]
Hi Lag-Llama team,

I would like to know if I can use this model for analyzing future customer trends based on feedback data from the past three months.

To clarify, I have customer feedback data for several products, including sentiments, emotions, comments, and issues, etc... categorized by city and state. Can this data be used to predict future trends in the coming months using this model?

Thanks in advance

**Comment by ashok-arjun** [comment]
Yes, possibly. Only trying it out with tell you how well it does on your tasks. Do note that right now, it only takes univariate inputs (i.e. a single variable) and forecasts that variable. So if you have N variables, it will forecast these N variables independently. It does not support covariates currently.
